sound/soc/sof/Makefile

```diff
@@ -1,7 +1,8 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o utils.o sof-audio.o stream-ipc.o ipc4.o
+		control.o trace.o utils.o sof-audio.o stream-ipc.o ipc4.o \
```

Comment:
Its not topology2.0 that mandates this change but rather IPC4, isnt it? Can you please update the commit message to reflect that?

Reply:
> Its not topology2.0 that mandates this change but rather IPC4, isnt it? Can you please update the commit message to reflect that?

Thanks @ranj063 I was confused by the relationship between topology2 and IPC4. I understand that there is no relationship between topology2 and IPC4 now. I will update the commit message then.